packages/react-component-library/src/components/Timeline/__tests__/Timeline.test.tsx

```diff
@@ -503,8 +502,8 @@ describe('Timeline', () => {
         <Timeline
```

Comment:
Looks like `consoleWarnSpy` can be removed further up.

Also would be good to update the name of the describe block now (currently `when `TimelineSide` is specified`).